app/forms/questionnaire_form.py

```diff
@@ -57,6 +57,17 @@ def __init__(
 
         super().__init__(**kwargs)
 
+    @property
+    def data(self):
+        d = super().data
+        try:
+            if d["csrf_token"] is None:
+                d["csrf_token"] = ""
+        except KeyError:
+            pass
+
+        return d
+
```

Comment:
`csrf_token` was being returned as `None` so have to convert specifically. Can't convert all cases of `None`s because some are tested as `None`....